extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -54,188 +34,268 @@
  *
  */
 
-public class PostFairDataToFairDataPoint extends Command{
+public class PostFairDataToFairDataPoint extends Command {
+
     private SimpleValueFactory f;
+    private static final Logger LOGGER = LoggerFactory.getLogger(PostFairDataToFairDataPoint.class);
 
-    /**
-     * This POST call should be called to POST FAIR data to a FAIR data Point.
-     * 
-     * This method also uploads the data to the configured resource.
-     * 
-     *  
-     *  This method uses the FAIRMetadata4j library.
-     * 
-     */
     @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+    public void doPost(HttpServletRequest req, HttpServletResponse res)
+            throws ServletException, IOException {
         f = SimpleValueFactory.getInstance();
-        
+
         ArrayList<IRI> datasetThemes = new ArrayList<IRI>();
         ArrayList<IRI> catalogThemes = new ArrayList<IRI>();
         String uuid_catalog = UUID.randomUUID().toString();
-        String uuid_dataset  = UUID.randomUUID().toString();
+        String uuid_dataset = UUID.randomUUID().toString();
         String uuid_distribution = UUID.randomUUID().toString();
         ArrayList<Literal> keyWords = new ArrayList<Literal>();
         Agent agent = new Agent();
         Date date = new Date();
-        
+
         String catalogString = null;
         String datasetString = null;
         String distributionString = null;
-        
-        Resource r = null;
 
+        Resource r = null;
 
         CatalogMetadata catalogMetadata = new CatalogMetadata();
         DatasetMetadata datasetMetadata = new DatasetMetadata();
         DistributionMetadata distributionMetadata = new DistributionMetadata();
-        
-        try{
+
+        try {
             StringBuffer jb = new StringBuffer();
             String line = null;
             BufferedReader reader = req.getReader();
             while ((line = reader.readLine()) != null)
-            jb.append(line);
-   
+                jb.append(line);
+
             JSONObject fdp = new JSONObject(jb.toString()).getJSONObject("metadata");
             JSONObject catalog = fdp.getJSONObject("catalog");
             JSONObject dataset = fdp.getJSONObject("dataset");
             JSONObject distribution = fdp.getJSONObject("distribution");
             JSONObject uploadConfiguration = fdp.getJSONObject("uploadConfiguration");
-            
-            if (!catalog.getBoolean("_exists")){
 
-    //          optional
-                try{
+            if (!catalog.getBoolean("_exists")) {
+
+                // optional
+                try {
                     catalogMetadata.setHomepage(f.createIRI(catalog.getString("http://xmlns.com/foaf/0.1/homepage")));
-                }catch(Exception ex){}
+                } catch (Exception ex) {
+                    LOGGER.warn(ex.getMessage());
+                }
```

Comment:
This goes for each `try/catch` in this method: how are errors communicated back to the front-end? Will the user know what went wrong is something fails?